quickstart.sh

```
@@ -0,0 +1,171 @@
+echo -e "\n\nSelect one option using up/down keys and enter to confirm:\n"
+
+select_option "${options[@]}"
+
+# Interactive menu for max steps using left/right arrows
+MIN_STEPS=3
+MAX_STEPS=20
+CUR_STEPS=5
+
+function adjust_steps_menu {
+    ESC=$( printf "\033")
+    cursor_blink_on()  { printf "$ESC[?25h"; }
```

Shellcheck (0.10.0)

[error] Use braces when expanding arrays, e.g. ${array[idx]} (or ${var}[.. to quiet).

(SC1087)